bindings/docstrings/nvdspreprocessmetadoc.h

```diff
@@ -7,29 +7,43 @@ namespace pydsdoc
         namespace NvDsPreProcessTensorMetaDoc
         {
             constexpr const char* descr = R"pyds(
-                Holds information about preprocessed tensor metadata
+                Holds information about preprocess tensor metadata in the preprocessed batch meta 
                 
-                :ivar raw_tensor_buffer: *void*, raw_tensor_buffer
-                :ivar buffer_size: *int*, buffer_size
-                :ivar tensor_shape: *vector*, tensor_shape
-                :ivar data_type: *NvDsDataType*, data_type
-                :ivar tensor_name: *string*, tensor_name
-                :ivar gpu_id: *int*, gpu_id
-                :ivar private_data: *void*, private_data
-                :ivar meta_id: *int*, meta_id)pyds";
+                :ivar raw_tensor_buffer: Holds raw tensor buffer preprocessed for infer.
+                :ivar buffer_size: *int*, Size of raw tensor buffer.
+                :ivar tensor_shape: *vector*, Shape of raw tensor buffer.
+                :ivar data_type: *NvDsDataType*, Model datatype for which tensor prepared.
+                :ivar tensor_name: *string*, Model input layer name.
+                :ivar gpu_id: *int*, gpu-id on which tensor prepared.
+                :ivar private_data: Private data used for the meta producer's internal memory management.)pyds";
```

Comment:
what is the type of private_data?

Reply:
nvidia skipped to mention type if it is void. private data is void type.